lib/robots/dor_repo/gis_delivery/load_vector.rb

```diff
@@ -21,31 +21,27 @@ def perform_work
 
           normalizer.with_normalized do |tmpdir|
             schema = Settings.geohydra.postgis.schema || 'druid'
-            # encoding =  # XXX: these are hardcoded encodings for certain druids -- these should be read from the metadata somewhere
-            #   case druid
-            #   when 'bt348dh6363', 'cc936tf6277'
-            #     'LATIN1'
-            #   else
-            #     'UTF-8'
-            #   end
 
             # sniff out shapefile from extraction
-            Dir.chdir(tmpdir)
-            shpfn = Dir.glob('*.shp').first
-            sqlfn = shpfn.gsub(/\.shp$/, '.sql')
-            errfn = 'shp2pgsql.err'
-            logger.debug "load-vector: #{bare_druid} is working on Shapefile: #{shpfn}"
+            shp_filename = Dir.glob("#{tmpdir}/*.shp").first
+            sql_filename = shp_filename.gsub(/\.shp$/, '.sql')
+            stderr_filename = "#{tmpdir}/shp2pgsql.err"
+
+            logger.debug "load-vector: #{bare_druid} is working on Shapefile: #{shp_filename}"
 
             # first try decoding with UTF-8 and if that fails use LATIN1
+            # see also https://github.com/sul-dlss/gis-robot-suite/issues/850
             begin
-              normalizer.run_shp2pgsql('4326', 'UTF-8', shpfn, schema, sqlfn, errfn)
-            rescue RuntimeError
-              normalizer.run_shp2pgsql('4326', 'LATIN1', shpfn, schema, sqlfn, errfn)
+              run_shp2pgsql('4326', 'UTF-8', shp_filename, schema, sql_filename, stderr_filename)
+            rescue RuntimeError => e
+              logger.warn("#{druid} -- fell through to LATIN1 encoding after calling run_shp2pgsql with " \
+                          "UTF-8 encoding and encountering error: #{e.message}; #{e.backtrace.join('; ')}")
```

Comment:
if we want to easily see whether this code path is used anymore, we could add a honeybadger alert here (a pattern we've used elsewhere).  since logs get rolled off after some number of days, an HB alert would make it easier to see whether this ever happens, without having to stay on top of the logs before they disappear.